### PR TITLE
[ML-6351] Update tensorflow to 1.13.1

### PR DESCRIPTION
**Pull request by lu-wang-dl**
Update tensorflow to 1.13.1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,11 +12,6 @@ RUN apt-get update && \
     apt-get install -y wget bzip2 openjdk-8-jdk unzip && \
     apt-get clean
 
-# Install protoc.
```
Comment (lu-wang-dl) [inline Dockerfile]: We install protocbuf in conda channel. This part is useless now.

**Comment by hanyucui** [comment]
I am not familiar with the use of protobuf here. The PR looks good other than protobuf-related changes.

**Comment by lu-wang-dl** [comment]
All of the protobuf-related changes belongs to tensorflow by calling `dev/update-tf-proto.sh`

**Comment by jkbradley** [comment]
LGTM too

**Pull request merged**